### PR TITLE
fix line exceeded maximum length

### PR DESCRIPTION
**Pull request by follitude**
PTAL @AndreMouche @zhangjinpeng1987

### CODE REVIEW

File: src/bin/tikv-server.rs [hunk 1]
```diff
@@ -552,11 +552,11 @@ fn get_rocksdb_cf_option(config: &toml::Value,
                                              Some(default_values.target_file_size_base));
     cf_opts.set_target_file_size_base(target_file_size_base as u64);
 
-    let level_zero_file_num_compaction_trigger =
+    let level_zero_file_num_compact_trigger =
```
Comment (siddontang) [inline src/bin/tikv-server.rs]: s/level_zero_file_num_compact_trigger/level_zero_compaction_trigger/

**Comment by follitude** [comment]
PTAL @siddontang

**Comment by siddontang** [comment]
LGTM

**Comment by siddontang** [comment]
#2086 already fixed it, we can close now.

**Pull request closed without merging**